authz_analyzer/datastores/aws/aws_ptrp_package/aws_ptrp/principals/principal.py

```diff
@@ -15,6 +15,12 @@
 regex_arn_account_id = re.compile(r"arn:aws:iam::([0-9]+):root$")
 
 
+def is_principal_root_account_id(
```

Comment:
why like this?
move it as a method of stmt_principal class and ask about the type